src/assets/minus.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 
 const Minus = (props) => {
-  let dimensions = (props.small ? 24 : 32)
- return <svg xmlns="http://www.w3.org/2000/svg" width={dimensions} height={dimensions} viewBox="0 0 45.724 10">
-    <path id="Path_27" data-name="Path 27" d="M0,0H45.724" transform="translate(0 5)" fill="none" stroke="#6098BF" strokeWidth="10"/>
-  </svg>
-}
+  let dimensions = props.large ? 30 : 24;
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width={dimensions} height={dimensions} viewBox="0 0 45.724 10">
+      <path
+        id="Path_27"
+        data-name="Path 27"
+        d="M0,0H45.724"
+        transform="translate(0 5)"
+        fill="none"
+        stroke="#6098BF"
```

Comment:
are the plus and minus icons meant to be two different colors? I think this should be #6bb4d6